doc/backing-store-crd.md

```diff
@@ -17,11 +18,23 @@ Adding support for a new type of backing-store is rather easy as it requires jus
 
 # Reconcile
 
-#### OBC type
+#### AWS-S3 type
 
-The operator will create a claim and the appropriate provisioner will create a new bucket or connect to existing one depending on the obc options. Once the claim is ready its details will be used to configure a cloud resource in NooBaa.
+Create a cloud storage agent within the NooBaa brain and use S3 API for storing encrypted chunks of data in the AWS cloud.
 
-#### PVC type
+#### S3-COMPATIBLE type
+
+Create a cloud storage agent within the NooBaa brain and use S3 API for storing encrypted chunks of data in any S3 API compatible endpoint.
+
+#### GOOGLE-CLOUD-STORAGE type
+
+Create a cloud storage agent within the NooBaa brain and use Google Cloud Storage API for storing encrypted chunks of data in Google Cloud Storage.
+
+#### AZURE-BLOB type
+
+Create a cloud storage agent within the NooBaa brain and use BLOB API for storing encrypted chunks of data in Azure cloud.
+
+#### PV-POOL type (Not yet implemented)
```

Comment:
Same as above 2

Also add the fact that its sts, and how should one scale it, what can be configured etc.